resources/views/Checkout/ConfirmationReinit.twig

```diff
@@ -0,0 +1,19 @@
+<script>
+    {% if success == "true" %}
+    OffAmazonPayments.initConfirmationFlow("{{ sellerId }}", "{{ amazonReferenceId }}", function (confirmationFlow) {
+        confirmationFlow.success();
+        doAmazonAuth();
+    });
+    {% else %}
+    OffAmazonPayments.initConfirmationFlow("{{ sellerId }}", "{{ amazonReferenceId }}", function (confirmationFlow) {
+        confirmationFlow.error();
+    });
+    {% endif %}
+    function doAmazonAuth () {
+        $.post('/payment/payone/checkout/doAuthFromOrder/{{ orderId }}', {
+            },
+            function (data, status) {
+            });
```

Comment:
Is there nothing else to do? Error handling or something like that?

Reply:
I removed the function. There is no need  for error handling because the Auth call has logging in the backend